java-example/README.MD

```diff
@@ -78,15 +83,10 @@ You can see the traces on the Traces Browser and the Application Map, and metric
 
 ### Create Charts with Metrics
 
-When the metrics data collected from the Wavefront Proxy are sent to our service, you can examine them in our user interface.
+When the metrics data collected from the Wavefront proxy are sent to our service, you can examine them in our user interface.
 
 Example:
 
 * The query `ts(jvm.threads.live)` shows the total number of live threads in the Petclinic application.
     Example:
     ![A screenshot showing the chart data when you query for ts(jvm.threads.live).](images/java_auto_instrumentation_metrics_chart.png)
-
-* This query `cumulativePercentile(85, mavg(15m, deriv(sum(ts(http.server.duration), le))))` shows the 85th percentile of HTTP response times over the last 15 minutes. In this case, 85 percent of the HTTP response times are 98.75ms or less.
```

Comment:
I am curious why you are removing this. In my opinion, it is helpful to explain WQL in plain english, but I guess it depends on who the audience is.